src/gsi/gsi_rfv3io_mod.f90

```diff
@@ -921,6 +1032,37 @@ subroutine read_fv3_netcdf_guess(fv3filenamegin)
           write(6,*)'fv3lam_io_dynmetvars2d_nouv is ',(trim(fv3lam_io_dynmetvars2d_nouv(i)),i=1,ntracerio3d)
       endif      
 
+    if (laeroana_fv3cmaq) then
+      jtracer = 0
+!name_chemvars3d chemguess from anainfo
+      do i=1,size(name_chemvars3d)
+        vartem=trim(name_chemvars3d(i))
+           if (ifindstrloc(aeronames_cmaq_fv3,trim(vartem)) > 0) then
+             jtracer=jtracer+1
+             fv3lam_io_tracerchemvars3d_nouv(jtracer)=trim(vartem)
+              write(6,*)'the chemvarname ',jtracer,vartem,' is found '
+           else
+              write(6,*)'the chemvarname ',vartem,' is not in aeronames_cmaq_fv3, !!!!!!!!!!'
```

Comment:
Should it crash/exit if a field is missing? If so, do we need to uncomment out the below lines?